src/scss/components/switch/mixins/_hxSwitchControl.scss

```diff
@@ -9,38 +9,49 @@
 
     &::-ms-check {
       border-color: $cyan-700;
+      border-color: var(--hxSwitchControl-focus-border-color, $cyan-700);
```

Comment:
Please use updated colors to replace `$cyan-700`.

Reply:
Changes have been done, please review.